src/modules/commander/Commander.cpp

```diff
@@ -2285,6 +2285,7 @@ Commander::run()
 						_in_flight_termination = true;
 
 						mavlink_log_critical(&mavlink_log_pub, "Attitude failure detected: force failsafe");
+						set_tune_override(TONE_PARACHUTE_RELEASE_TUNE);
```

Comment:
Do I understand it correctly that this is only printed and played once because of `_in_flight_termination`?